src/step_definitions/performance.js

```diff
@@ -43,7 +46,25 @@ defineSupportCode(function ({ When }) {
     });
 
     return browser.driver.wait(() => {
-      return proxyDone;
+      return (
+        this.performanceReportFile = uniqueFileName,
+        proxyDone
+      )
     });
   });
+
+  Then(/^the requests should take a maximum of "([^"]*)" milliseconds$/, function (maxTiming) {
+    const maxTimingInt = parseFloat(maxTiming);
+    const slowRequests = analyser.checkTiming(this.performanceReportFile, maxTimingInt);
```

Comment:
You have to put it in try catch block. You're not catching an error you throw and it will result in instant exit from kakunin process - it won't test anything after this step.

Reply:
Added try catch block

Reply:
you have to put slowRequests in try-catch not outside